patches/master_patch.patch

```diff
@@ -106,6 +110,26 @@ index 070d7c6008b1df85486014c538bc6e98b6771220..c26cad1e37ea14fabab1ff39c348cf1f
  }
  
  assert(!is_ios, "Chromium/iOS shouldn't use anything in //chrome")
+@@ -957,6 +962,9 @@ if (is_win) {
+ 
+     widevine_sign_file("sign_cdm_adapter_for_widevine") {
+       file = "$root_out_dir/$widevine_cdm_path/widevinecdmadapter.plugin"
++      if (!muon_chromium_build) {
++        signature_file = "$file.dummy"
```

Comment:
I don't understand why this is necessary. You can pass the signature file in `widevine_sign_file`

Reply:
^^ we can create our own target instead of using chrome's

Reply:
That was meant for preventing duplicate output

Reply:
```
ERROR at //third_party/widevine/cdm/widevine.gni:15:3: Duplicate output file.
  action(target_name) {
  ^--------------------
Two or more targets generate the same output:
  WidevineCdm/_platform_specific/mac_x64/widevinecdmadapter.plugin.sig

This is can often be fixed by changing one of the target names, or by
setting an output_name on one of them.

Collisions:
  //chrome:sign_cdm_adapter_for_widevine
  //electron/app/mac:sign_cdm_adapter_for_widevine_on_brave

See //third_party/widevine/cdm/widevine.gni:15:3: Collision.
  action(target_name) {
  ^--------------------
```

Reply:
so why is the signature file set to $file.dummy?

Reply:
can we just use all the same params as chrome?

Reply:
to make it different from `WidevineCdm/_platform_specific/mac_x64/widevinecdmadapter.plugin.sig`
cause we don't use it here. we have our own `//electron/app/mac:sign_cdm_adapter_for_widevine_on_brave`